src/modules/nRF24/nRF24.cpp

```diff
@@ -230,14 +230,9 @@ int16_t nRF24::setFrequency(int16_t freq) {
     return(ERR_INVALID_FREQUENCY);
   }
 
-  // set mode to standby
-  int16_t state = standby();
```

Comment:
Seems that there's no need to put modem in standby to change frequency; 
I've tested it and double checked on other nRF24 drivers.